gittrigger/gittrigger.go

```diff
@@ -35,12 +35,36 @@ func (g *gitTrigger) getLatestCommit() error {
 	return nil
 }
 
-func (g gitTrigger) checkTrigger() bool {
+func (g *gitTrigger) GetRemote() (string, error) {
+	var out bytes.Buffer
+	cmd := exec.Command("git", "remote")
```

Comment:
This can return multiple lines if you have more than one remote.

Reply:
Thanks, good catch

Reply:
Fixed in 1b02355